docs/getting_started.rst

```diff
@@ -1,9 +1,31 @@
 .. include:: references.txt
 
 .. _gs:
+.. doctest-skip-all
```

Comment:
does linking to the `gs` label still work if you put the `.. doctest-skip-all` between it and the heading?
Even if yes, I'd suggest to exchange the order of those two lines.